Sources/SwiftGenKit/Utils/Metadata.swift

```diff
@@ -57,14 +57,15 @@ enum Metadata {
     }
   }
 
-  private static func describe(dictionary: [String: Any]) -> [[String: Any]] {
-    return dictionary.sorted {
-      $0.key < $1.key
-    }.map { item -> [String: Any] in
-      Metadata.generate(for: item.value).merging([
-        Key.name: item.key
-      ]) { _, rhs in rhs }
-    }
+  private static func describe(dictionary: [String: Any]) -> [String: Any] {
+    return Dictionary.init(uniqueKeysWithValues: dictionary.map { item in
```

Comment:
[CodeStyle nitpick] Don't we have a SwiftLint rule to avoid `T.init(…)` and use `T(…)` instead?
I know typing `.init(` sometimes helps the autocomplete engine, but in the end code I think it can go away

Reply:
Ehm, yeah, probably from when Xcode autocomplete was being buggy (again).